src/packages.builds

```diff
@@ -10,7 +10,7 @@
     <Project Include="Native\pkg\**\*.builds">
       <AdditionalProperties>$(AdditionalProperties);SkipCreatePackageOnMissingFiles=true</AdditionalProperties>
     </Project>
-    <Project Include="*\pkg\*.builds">
+    <Project Include="*\pkg\*.builds" Condition="'$(Native)' != 'true'">
```

Comment:
Agreed.  It's probably fine for the command-line switch, but not as a property name.